starWarsScene.js

```diff
@@ -12,18 +13,18 @@ function seeRebelsEscape(rebels) {
   var escapedRebelCalls = []
   var i = 0
 
-  do {
+  while (rebels[i] && rebels[i]['plotArmor']) {
     escapedRebelCalls.push('Oh no! They\'re getting away!')
     i++
-  } while (rebels[i] && rebels[i]['plotArmor'])
+  }
 
   return escapedRebelCalls
 }
 
 function shootAtNothing() {
   var shotsAtNothing = []
 
-  for (var i = 1; i < 4; i++) {
+  for (var i = 1; i < 5; i++) {
```

Comment:
Nicely done. Could have also gone from `0` to `< 4` or from `1` to `<= 4`. Any are correct.